apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.js

```diff
@@ -1,27 +1,42 @@
 // @ts-check
-const ERR = require('async-stacktrace');
 import * as express from 'express';
-import * as path from 'path';
-const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const asyncHandler = require('express-async-handler');
+import { z } from 'zod';
+import * as sqldb from '@prairielearn/postgres';
 
 import { config } from '../../lib/config';
-import * as sqldb from '@prairielearn/postgres';
 
 const router = express.Router();
 const sql = sqldb.loadSqlEquiv(__filename);
 
-router.get('/', function (req, res, next) {
-  debug('GET /');
-  var params = {
-    assessment_id: res.locals.assessment.id,
-    link_exam_id: config.syncExamIdAccessRules,
-  };
-  sqldb.query(sql.assessment_access_rules, params, function (err, result) {
-    if (ERR(err, next)) return;
-    res.locals.access_rules = result.rows;
-    debug('render page');
-    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
-  });
+const assessmentAccessRulesSchema = z.object({
```

Comment:
Schemas conventionally start with a capital letter.

Reply:
I have updated this to a capital letter in commit [ac859c4](https://github.com/PrairieLearn/PrairieLearn/pull/9072/commits/ac859c49a2700c2d605b759748efd2889f5c42d8).